### ISSUE TITLE
DTS-M1 antenna

### ISSUE THREAD

**Issue by NathanKell** [KSP-RO/RealismOverhaul#1700]
@PhineasFreak I noticed this got rebalanced last fall. What's its intended purpose now? At 170Gm it looks like it can only do Mercury or Venus-Conjunction, it's too short even for Venus opposition, but it has a high dish angle compared to most IP dishes.

**Comment by PhineasFreak** [comment]
I remember it being super OP and i nerfed it, probably too much as it seems...two of them would make an excellent combo (wide cone angle and high connection range) though.

Feel free to rebalance it as required, i am no expert on that stuff.

**Comment by NathanKell** [comment]
Ah, yeah, it was fairly OP as I recall.

It and the HG-5 make good 'second tier' Venus and Mars dishes if we tweak the ranges slightly. But that still leaves me wondering what the use case for the HG-20 is. Perhaps a 3rd generation near-planets dish, if it gets an even 400Gm range? Stick it in the same node as Voyager, maybe, @pap1723 ? I'll do the range changes tonight.

**Comment by PhineasFreak** [comment]
My personal feel is that we currently have more early and small dishes than late and large. It makes balancing somewhat difficult. But, again, it may be just me...

**Comment by NathanKell** [comment]
Yep! Although thanks to 1.2 we have some good late dishes too...
We could make the HG-20 or DTS a really big dish too maybe if you think
there's a hole?

**Comment by PhineasFreak** [comment]
I do not think that there is a hole. We have 3 stock dish antennas (DTS-M1, HG-55, HG-5) plus the VSR one (HG-20). More than enough antennas to balance in order cover the inner and some of the outer (Mars, Ceres, Vesta) planets.

We could have some early retractable (Ranger & Mariner style, inefficient and slow) and some later (LRO & BepiColombo style, lower power consumption, higher range and better data rates)

The data rate is also a fine way to balance similar (or not) antennas. Between two antennas, with the same overall construction, one could have been an early one (analog transmittions, low data rates) while the other could be more advanced (digital transmittions, high data rates).

**Comment by pap1723** [comment]
This is exactly what NathanKell and I were talking about right before he
went to bed tonight. You seem to have a very good grasp on this Phineas,
would you care to put together a sheet of what antennas with what ranges
and transfer speeds / costs you think we should have? I can add them as
parts easily enough.

On Jul 11, 2017 2:51 AM, "Phineas Freak" <notifications@github.com> wrote:

I do not think that there is a hole. We have 3 stock dish antennas (DTS-M1,
HG-55, HG-5) plus the VSR one (HG-20). More than enough antennas to balance
in order cover the inner and some of the outer (Mars, Ceres, Vesta) planets.

We could have some early retractable (Ranger & Mariner style, inefficient
and slow) and some later (LRO & BepiColombo style, lower power consumption,
higher range and better data rates)

The data rate is also a fine way to balance similar (or not) antennas.
Between two antennas, with the same overall construction, one could have
been an early one (analog transmittions, low data rates) while the other
could be more advanced (digital transmittions, high data rates).

—
You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub
<https://github.com/KSP-RO/RealismOverhaul/issues/1700#issuecomment-314363795>,
or mute the thread

**Comment by PhineasFreak** [comment]
@pap1723 I could make a table with the current properties to see if they are balanced but the problem is that i have not actually played with RP-0 since...ever. So, i do not know if one change in the range is good or another change in the data rate is bad (in the context of a career game). I trust that you and @NathanKell will have a better feeling about the overall balancing.

Another thing is that these antennas were configured with the old RP-0 tree in mind (much more "compressed") so jumping from one antenna to another was not as difficult as it is now.

**Comment by pap1723** [comment]
@PhineasFreak That is understandable! Very surprised to hear you never played RP-0! I will focus on figuring out the antennas and will report back.

**Comment by PhineasFreak** [comment]
Well, not never but i skipped some (KSP 1.1.X & 1.2.X) versions. Configuring parts for RO is very time consuming and by the end of the day it has sucked all the fun out of the game (the CMES and Coatl Aerospace nightmares are still fresh...). Add 5+ years of burnout, plus fighting all these KSP updates, and you get a very nice package!

At least it is good to see and hear that the next RP-0 version is the best one ever. You guys/gals are doing an outstanding job!

**Comment by PhineasFreak** [comment]
@NathanKell and @pap1723 I forgot to comment on the larger dish antennas: they are are more or less modeled after:

* **RA-2:** OSIRIS-REx, MAVEN, MRO, Ulysses antennas.
* **RA-150:** Pioneer 10/11 antennas.
* **RA-100:** Voyager 1/2 antennas.
* **Communotron 88-88:** Galileo antenna.

These only go after the part general sizing, part appearance and effective range. Data rates are balanced between each other (though similarly to the real life data rates from these spacecraft) and cone angles after the apparent dish diameter and nominal operational frequency (S and X bands).

Note that data rates are also modeled after computer data rates: 2 Mbps means a rate of 2.048. If you feel that some antennas have a low data rate then you can increase it by following the powers of 2 (data interval is always set as 1).

**Comment by NathanKell** [comment]
Ah, those overlap with the RT antennas no? Are there reference antennas such that they wouldn't overlap?

**Comment by PhineasFreak** [comment]
You cannot really set the antennas not to overlap. There are definite targets in the solar system, mainly set by the distance from the Earth and lesser by the categorization (inner & outer planets). Large dishes (Pioneer & Voyager style) will always have a high enough range for even "interstellar" space, while smaller dishes will have to be distributed according to the era and more specific targets (Venus, Mars, Jupiter, Saturn). Also, advances in technology can make antennas with identical mass/range stats more attractive if they have lower standby/transmittion consumption and higher data rates.

Would it be too bad if two antennas have exactly (or close enough) stats with a different overall format? I think of it like the choosing an engine: we have a lot of them that are the same (e.g. LR-79, LR-89) and the user can choose the one that is a better fit.

**Comment by NathanKell** [comment]
Overlap, yeah, I know you can't. But they're literally identical stats was my point, aren't they? At least they can, as you say, represent different tech levels...LR79 and 89 are in fact even more different than I was thinking because they have different reliability, cost, and upgrade paths...

Also, hit another issue, a major one: last February, it looks like, you lowered the max temps of the antennas to 473K across the board. That means, in Deadly Reentry, a max temp (before they start melting/burning/dying) of ~410K. That's only about 140C, and makes a Venus mission, let alone a Mercury mission, impossible as far as I can tell. Can you explain the reasoning for lowering their max temps? Most of our science and probe parts have a max temp of ~1073K, which makes Mercury missions perfectly possible.

**Comment by PhineasFreak** [comment]
The temperature changes are because:

* I forgot about the DRE nerf
* I have dropped the ball regarding my overall quality assurance lately

So either you or @pap1723 could increase their max temps by 100 - 150 K to ensure that the parts will work for that environments.

**Comment by NathanKell** [comment]
No problem at all! Just wanted to check to see if I was missing something with the reconfigure. :)

I'll do a pass this evening.

**Comment by PhineasFreak** [comment]
Semi-offtopic but it seems that the next version of RT will support timewarp transmittions: https://github.com/RemoteTechnologiesGroup/RemoteTech/commit/6e61d130e0d041865f1596ed4ec485b80a6fb263

So that lower data rates would not be so much of a hassle anymore (256 bit/s Pioneer antennas anyone?)

**Comment by NathanKell** [comment]
Yep been begging taxiservice on slack about it and happily another RTer had
a better idea than mine and submitted the pr. Hurrah!

**Comment by DRVeyl** [comment]
Over 4 years later.  This probably closes just on its own, or possibly as a result of <https://github.com/KSP-RO/RealismOverhaul/issues/1992>
